### DISCUSSION TITLE
Next.js and firebase

### DISCUSSION THREAD

**Discussion by imsamimalik** [firebase/firebase-js-sdk#5425] [General]
I'm importing firebase from firebase/app but error occurs saying firebase is not defined. 

`import firebase from "firebase/app";

console.log(firebase);
`
Error: 
> ReferenceError: firebase is not defined

**Comment by Feiyang1** [comment]
Are you using `9.0.0`? `9.0.0` has a completely updated API, and is not compatible with v8 API. You can check out the new [get started guide](https://firebase.google.com/docs/web/setup) to learn about it.
